lib/opentok/version.rb

```diff
@@ -1,4 +1,4 @@
 module OpenTok
   # @private
-  VERSION = '4.3.0'
+  VERSION = '4.2.0'
```

Comment:
@superchilled I'm worried a bit if this part is fine. Looks like version downgrade right now

Reply:
Current version is 4.2.0 in `main`, see: https://github.com/opentok/OpenTok-Ruby-SDK/blob/main/lib/opentok/version.rb

The new version *will* be 4.3.0, but will bump this in a separate PR (this current PR will be merged into `dev`, then when we are ready to release the new version of the Gem, will merge `dev` into `main` with a version bump).

Reply:
got it, thanks 👍